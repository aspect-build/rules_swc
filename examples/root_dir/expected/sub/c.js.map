{"version":3,"sources":["c.ts"],"sourceRoot":"","sourcesContent":["export const c: string = \"c\";\n"],"names":["c"],"mappings":"AAAA,OAAO,IAAMA,IAAY,IAAI"}