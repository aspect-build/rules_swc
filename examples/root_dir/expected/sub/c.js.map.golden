{
    "sources": [
        "../../src/sub/c.ts"
    ]
}